codemeta.json

```diff
@@ -0,0 +1,72 @@
+{
+  "@context": "https://raw.githubusercontent.com/codemeta/codemeta/master/codemeta.jsonld",
+  "@type": "Code",
+  "author": [
+    {
+      "@id": "https://orcid.org/0000-0002-6620-807X",
+      "@type": "Person",
+      "email": "markus.ankenbrand@uni-wuerzburg.de",
+      "name": "Markus J. Ankenbrand",
+      "affiliation": "Department of Animal Ecology and Tropical Biology (Zoology III), University of Würzburg, Germany"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Simon Pfaff",
+      "affiliation": "Department of Bioinformatics, University of Würzburg, Germany"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Niklas Terhoeven",
+      "affiliation": "Center for Computational and Theoretical Biology, University of Würzburg"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Musga Qureischi",
+      "affiliation": "Centre for Experimental Molecular Medicine, University Clinics Würzburg, Germany"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Maik Guendel",
+      "affiliation": ""
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Clemens L. Weiss",
+      "affiliation": "Research Group for Ancient Genomics and Evolution, Department of Molecular Biology, Max Planck Institute for Developmental Biology, Tübingen, Germany"
+    },
+    {
+      "@id": "https://orcid.org/0000-0002-0022-320X",
+      "@type": "Person",
+      "email": "",
+      "name": "Thomas Hackl",
+      "affiliation": "Department of Civil and Environmental Engineering, Massachusetts Institute of Technology"
+    },
+    {
+      "@id": "https://orcid.org/0000-0003-4166-5423",
+      "@type": "Person",
+      "email": "frank.foerster@ime.fraunhofer.de",
+      "name": "Frank Förster",
+      "affiliation": "Fraunhofer Institute for Molecular Biology and Applied Ecology IME, Applied Ecology and Bioresources, Gießen, Germany"
+    }
+  ],
+  "identifier": "https://doi.org/10.5281/zenodo.883594",
+  "codeRepository": "https://github.com/chloroExtractorTeam/chloroExtractor",
+  "datePublished": "2017-09-26",
```

Comment:
Do we need to update the dates or will that be done by JOSS?

Reply:
I don't think this will be automatically updated. I guess we have to do this ourselves. Maybe we can re-run the generation script...

Reply:
Thats okay for me, but in that case I would postpone the pull request until we will submit as last or last before last commit. Or you adjust the dates for tomorrow, as we want to submit it tomorrow.

Reply:
You are right we should probably wait.
We have another chicken-or-egg problem: We want to have the doi for version 1.0.0 in this file but we do not have this doi until we release version 1.0.0 ;-)
It is probably best to use the general concept doi of zenodo then: https://doi.org/10.5281/zenodo.883594

Reply:
Won't be the optimal but a good and solid solution or do we get an DOI for draft releases?

Reply:
Another possibility is, deactivation of automated zenodo service, reserving a DOI at zenodo, and making the release and preparing a ZIP-File of that release (does the release tars/zips now include recursive checkouts?) and publish that on our own upload with the reserved DOI. In that case we know the DOI before making the release.